docs/lms_index.rst

```diff
@@ -0,0 +1,24 @@
+lms
+***
+
+The ``lms`` directory in edx-platform is home to the code needed for the LMS
+(Learning Management System) which is not also needed for the Course Authoring
+Studio.
+
+.. toctree::
+    :maxdepth: 2
+
+    lms/modules
+    lms/djangoapps/badges/modules
+    lms/djangoapps/branding/modules
+    lms/djangoapps/bulk_email/modules
+    lms/djangoapps/courseware/modules
+    lms/djangoapps/coursewarehistoryextended/modules
+    lms/djangoapps/email_marketing/modules
+    lms/djangoapps/experiments/modules
+    lms/djangoapps/lti_provider/modules
+    lms/djangoapps/mobile_api/modules
+    lms/djangoapps/notes/modules
+    lms/djangoapps/rss_proxy/modules
+    lms/djangoapps/shoppingcart/modules
+    lms/djangoapps/survey/modules
```

Comment:
How will we ensure that new source directories get added to this process?

Reply:
These are only here in the first place as a hack to work around the fact that they can't be cleanly imported as, for example, `lms.djangoapps.survey` instead of just `survey`.  Most new apps (and we're trying not to add many more) should be importable under the full package path as we try to get rid of this hack to the PYTHONPATH.